linera-execution/src/evm/revm.rs

```diff
@@ -223,12 +260,16 @@ impl UserServiceModule for EvmServiceModule {
 // functionalities accessed from the EVM.
 const PRECOMPILE_ADDRESS: Address = address!("000000000000000000000000000000000000000b");
 
-fn u8_slice_to_application_id(vec: &[u8]) -> ApplicationId {
+fn u8_slice_to_cryptohash(vec: &[u8]) -> CryptoHash {
```

Comment:
There's already an `impl TryFrom<&[u8]> for CryptoHash`.

Reply:
Ok, thank you. Missed that.
I restructured the code accordingly.